src/v3/src/components/Link/Link.tsx

```diff
@@ -88,6 +88,8 @@ const Link: UISchemaElementComponent<{
           aria-describedby={ariaDescribedBy}
           data-se={dataSe}
           target={target}
+          // eslint-disable-next-line react/jsx-props-no-spreading
+          {...(target === '_blank' && { rel: 'noopener noreferrer' })}
```

Comment:
added these attributes to any links that have target="_blank" for security. This will prevent possible redirects to a malicious URL when target is set to _blank.
Reference: https://mui.com/material-ui/react-link/#security